Dockerfile

```diff
@@ -1,20 +1,53 @@
-FROM node:10-alpine
+FROM alpine:3.10 AS buildcontainer
 
-RUN apk update && apk upgrade && apk add nmap nmap-scripts
+ENV  NMAP_SHA256SUM="fcfa5a0e42099e12e4bf7a68ebe6fde05553383a682e816a7ec9256ab4773faa" \
```

Comment:
This is a pit nitpicky but these should rather be `ARG`s, so that they can be changed from outside if required.